docs/source/conf.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# sys.path.insert(0, os.path.abspath('./'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.intersphinx',
+              'sphinx.ext.viewcode',
+              'sphinx.ext.autodoc',
+              'sphinx.ext.autosectionlabel',
+              'autodocsumm',
+              # 'sphinx_automodapi.automodapi',
+              'sphinx.ext.napoleon',
+              # 'sphinxcontrib.autoprogram'
+              ]
+
+autosectionlabel_prefix_document = True
+autosectionlabel_maxdepth = 2
+autosummary_generate = True
+autosummary_generate_overwrite = True
+# numpydoc settings
+numpydoc_show_class_members = True
+numpydoc_show_inherited_class_members = True
+numpydoc_attributes_as_param_list = False
+numpydoc_class_members_toctree = False
+
+# Napoleon settings
```

Comment:
Napoleon is not necessary, we don't use lumpy enough and pibooth don't use the Google style docstrings